src/urllib3/response.py

```diff
@@ -321,8 +321,8 @@ def isclosed(self):
         return self.closed
 
     # Compatibility methods for http.client.HTTPResponse
-    def getheaders(self) -> typing.MutableMapping[str, str]:
-        return self.headers
+    def getheaders(self) -> typing.List[typing.Tuple[str, str]]:
+        return list(self.headers.items())
```

Comment:
Mentioned that breaking change in https://github.com/urllib3/urllib3/issues/2065#issuecomment-731951335 for lack of a better place. We should probably start using towncrier :)

Reply:
Thanks for adding that. I'm open to using towncrier for tracking changes, can create an issue for this work.

Reply:
I think something that might be valuable here is creating annotation aliases. For example `HeaderPair = typing.Tuple[str, str]` it makes the annotations clearer when reading the code and when reusing that `Tupe[str, str]` pattern for Headers as it conveys intent